pipelines/ONT/VariantCalling/ONTWholeGenome.wdl

```diff
@@ -14,6 +8,31 @@ import "../../../tasks/Utility/Finalize.wdl" as FF
 import "../../../tasks/QC/SampleLevelAlignedMetrics.wdl" as COV
 
 workflow ONTWholeGenome {
+
+    meta {
+        description: "A workflow that performs single sample variant calling on Oxford Nanopore reads from one or more flow cells. The workflow merges multiple samples into a single BAM prior to variant calling."
```

Comment:
Typo: not `multiple samples`, but `multiple flowcells`

Reply:
All. I don't think we have multi-sample workflows except two or three for joint calling.